frontend/app/src/main/java/de/amosproj3/ziofa/ui/configuration/data/ConfigurationScreenState.kt

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2024 Luca Bretting <luca.bretting@fau.de>
+//
+// SPDX-License-Identifier: MIT
+
+package de.amosproj3.ziofa.ui.configuration.data
+
+sealed class ConfigurationScreenState {
+    data class LIST(val options: List<EBpfProgramOption>) : ConfigurationScreenState()
+
+    data class ERROR(val errorMessage: String) : ConfigurationScreenState()
+
+    data object LOADING : ConfigurationScreenState()
```

Comment:
This should be similar to `ConfigurationUpdate`, either I would prefer 'Valid, Invalid, Loading` so the naming is consistent. Otherwise we could use `Success, Failure` like in kotlin's Result class.